compiler/z/codegen/S390Instruction.cpp

```diff
@@ -2457,6 +2468,25 @@ TR::S390RILInstruction::generateBinaryEncoding()
             #endif
             }
          }
+#elif defined(TR_TARGET_64BIT) && defined(J9ZOS390)
+      if (comp->getOption(TR_EnableTrampolines) && comp->getCodeCacheSwitched())
+         {
+         TR::SymbolReference *calleeSymRef = NULL;
+
+         calleeSymRef = getSymbolReference();
+
+         if (calleeSymRef != NULL)
+            {
+            if (calleeSymRef->getReferenceNumber() >= TR_S390numRuntimeHelpers)
+               cg()->fe()->reserveTrampolineIfNecessary(comp, calleeSymRef, true);
+            }
+         else
+            {
+               #ifdef DEBUG
+               printf("Missing possible re-reservation for trampolines.\n");
+               #endif
```

Comment:
Extra indentation here on `printf` and the preprocessor macros should not be indented at all.